Sources/MapboxMaps/Foundation/Extensions/Core/MapSnapshotOptions.swift

```diff
@@ -13,6 +13,12 @@ public struct MapSnapshotOptions {
     /// Resource fetching options to be used by the snapshotter.
     public let resourceOptions: ResourceOptions
 
+    /// Flag that determines if the logo should be shown on the snapshot
+    public let showsLogo: Bool
+
+    /// Flag that determines if attribution should be shown on the snapshot
+    public let showsAttribution: Bool
```

Comment:
is this flag unused?

Reply:
I have put this PR up, but will not merge until I rebase off of Julian's Snapshotter work.

Reply:
let's mark it as a draft in that case